docs/cwl/conformance.rst

```diff
@@ -0,0 +1,93 @@
+.. _cwlConformance:
+
+Toil CWL Conformance v1.2.1
+===========================
+
+Note: To fully support `inplace_update`, ``--bypass-file-store`` should be enabled.
+See :ref:`inplaceupdaterequirement`.
+
+The following conformance tests are tested against the latest version of master:
```

Comment:
These aren't the individual conformance tests, these are particular features. And "the latest version of master" might only make sense to us.

```suggestion
The badges below show Toil's support for different aspects of the CWL specification. They describe the current development version of Toil, not necessarily the release documented here.
```

If we put a version of the spec here, it might stop being accurate when we update the spec we test against and the badges change but the old docs don't.